pkg/lint/rules/template.go

```diff
@@ -55,11 +55,7 @@ func Templates(linter *support.Linter, values map[string]interface{}, namespace
 	// Load chart and parse templates
 	chart, err := loader.Load(linter.ChartDir)
 
-	chartLoaded := linter.RunLinterRule(support.ErrorSev, fpath, err)
-
-	if !chartLoaded {
-		return
-	}
```

Comment:
Unfortunately, this is not going to work. If the chart did not load a message is added to the linters messages. Since the chart could not be loaded the other linting tests will also fail. There is no chart to operate on. So, it needs to return.

Looking through the times this function returns, it only happens when something that blocks future steps from happening is encountered. This return should not be removed.

Reply:
Yes, I also found this problem. if there is an error in `Chart.yaml`, this check function should return.

> Looking through the times this function returns, it only happens when something that blocks future steps from happening is encountered. This return should not be removed.

The return time of the function is fine. Maybe this is in line with the design.